packages/core/logger/src/Logger.js

```diff
@@ -176,6 +174,28 @@ export function patchConsole() {
   consolePatched = true;
 }
 
+export function unpatchConsole() {
```

Comment:
can `patchConsole` return a Disposable [0] that does this instead?

[0] https://github.com/parcel-bundler/parcel/blob/b777024226c4f01efb30a2265dd45d90ae801a4d/packages/utils/events/src/Disposable.js

Reply:
I guess so but it wouldn't really change anything would it?